containers/iam-reana-sync/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG BASETAG=0.9.1-alpha.3
+ 
+# To be changed to specific version+hsa
+FROM reanahub/reana-server:${BASETAG}
+LABEL maintainer="E. Gazzarrini"
+
+# Workdir is /home
+COPY requirements.txt add_reana_users.py generate_email_list.py iam.ini /home/
```

Comment:
`iam.ini` file is missing. At least from the `containers/iam-reana-sync` dir. missing or should be added as a secret ? (not sure what should include)